Reexport the other packages (#46)
QEDjl-project/QED.jl: docs/src/index.md

> # QED

Reviewer: ```suggestion
# QED.jl
```

> This is the documentation for [QED](https://github.com/QEDjl-project/QED.jl). It represents the combination of the following subpackages:

Reviewer: ```suggestion
This is the documentation for [`QED.jl`](https://github.com/QEDjl-project/QED.jl). It represents the combination of the following subpackages:
```

**The two fundamental packages**:

> - [QEDbase](https://github.com/QEDjl-project/QEDbase.jl): Interfaces and some functionality on abstract types. [Docs](https://qedjl-project.github.io/QEDbase.jl/stable/)

Reviewer: ```suggestion
- [`QEDbase.jl`](https://github.com/QEDjl-project/QEDbase.jl): Interfaces and some functionality on abstract types. [Docs](https://qedjl-project.github.io/QEDbase.jl/stable/)
```

> - [QEDcore](https://github.com/QEDjl-project/QEDcore.jl): Implementation of core functionality that is needed across all or most content packages. [Docs](https://qedjl-project.github.io/QEDcore.jl/stable/)

Reviewer: ```suggestion
- [`QEDcore.jl`](https://github.com/QEDjl-project/QEDcore.jl): Implementation of core functionality that is needed across all or most content packages. [Docs](https://qedjl-project.github.io/QEDcore.jl/stable/)
```

**The content packages**:

> - [QEDprocesses](https://github.com/QEDjl-project/QEDprocesses.jl): Scattering process definitions, models, and calculation of cross-sections and probabilities. [Docs](https://qedjl-project.github.io/QEDprocesses.jl/stable/)

Reviewer: ```suggestion
- [`QEDprocesses.jl`](https://github.com/QEDjl-project/QEDprocesses.jl): Scattering process definitions, models, and calculation of cross-sections and probabilities. [Docs](https://qedjl-project.github.io/QEDprocesses.jl/stable/)
```

> - [QEDevents](https://github.com/QEDjl-project/QEDevents.jl): Generation of random events for scattering processes. [Docs](https://qedjl-project.github.io/QEDevents.jl/stable/)

Reviewer: ```suggestion
- [`QEDevents.jl`](https://github.com/QEDjl-project/QEDevents.jl): Monte-Carlo event generation for scattering processes. [Docs](https://qedjl-project.github.io/QEDevents.jl/stable/)
```

> - [QEDfields](https://github.com/QEDjl-project/QEDfields.jl): Representation of background fields. [Docs](https://qedjl-project.github.io/QEDfields.jl/stable/)

Reviewer: ```suggestion
- [`QEDfields.jl`](https://github.com/QEDjl-project/QEDfields.jl): Description of classical electromagnetic fields used in background-field approximations. [Docs](https://qedjl-project.github.io/QEDfields.jl/stable/)
```
